src/styles/_header.scss

```diff
@@ -294,46 +299,48 @@
     //language buttons that look like links
     .usa-button--unstyled {
       @include u-text('accent-cool-lighter');
+      @include u-bg('base-dark');
       text-decoration: none;
+      line-height: 25px;
     }
+
     //right-to-left language buttons in dropdown menu
     .banner__language-selector-item--rtl {
       @include language-button--hover('primary');
       display: flex;
       flex-direction: row-reverse;
-      @include active-border(
-        'right',
-        'primary',
-        'accent-cool-lighter',
-        'base-dark'
-      );
     }
     //left-to-right language buttons in dropdown menu
     .banner__language-selector-item--ltr {
       @include language-button--hover('primary');
-      @include active-border(
-        'left',
-        'primary',
-        'accent-cool-lighter',
-        'base-dark'
-      );
     }
     //all language buttons when there are 5 or fewer languages
     .banner__language-selector-button-group {
       @include flex-center-space-between();
       .banner__language-selector-item {
         @include language-button--hover('primary');
-        @include active-border(
-          'top',
-          'primary',
-          'accent-cool-lighter',
-          'base-dark'
-        );
         margin: 0 units('105');
         &:hover {
           padding-top: 0;
         }
       }
     }
   }
+  // to indicate which language is in use or active, there is a bar on one side on one side of the language button
+  // left-to-right languages active border
+  .banner__language-selector-item--ltr.banner__language-selector-item--active {
```

Comment:
Does this count as 2 classes or 1 class?

Reply:
I think 2 but they both apply

Reply:
Interesting.